src/ModelContextProtocol.AspNetCore/README.md

```diff
@@ -0,0 +1,54 @@
+# MCP C# ASP.NET Core SDK
+
+[![NuGet preview version](https://img.shields.io/nuget/vpre/ModelContextProtocol.svg)](https://www.nuget.org/packages/ModelContextProtocol/absoluteLatest)
```

Comment:
@stephentoub I wonder if the introduction of a new package changes our assumption about whether the package READMEs should be kept from the repo README.

Reply:
In the future it might make sense to publish all samples via docfx, since it can also ensure that the snippets type check. Then all README.md's can point to those. How does that sound?

Reply:
Sample code directly in the NuGet README and repo README improves the "walk-up" experience. Always a tradeoff to be balanced between that experience and ensuring the samples are provably correct/updated.

We'll inevitably have to do _something_, but for the time-being I concur we should leave the main package's readme as is pulling in the repo readme, with other packages have tailored content.